web-tier/src/main/java/com/outdoor/foodcalc/service/DishService.java

```diff
@@ -47,12 +47,12 @@ public List<CategoryWithDishes> getAllDishes() {
         return categories.stream()
                 .map(c -> {
                     final CategoryWithDishes cView = new CategoryWithDishes();
-                    cView.id = c.id;
-                    cView.name = c.name;
-                    final List<Dish> dishList = dishesMap.get(c.id);
-                    cView.dishes = (dishList == null) ? new ArrayList<>() : dishList.stream()
+                    cView.setId(c.getId());
```

Comment:
use Builder for building complex objects

Reply:
Builder added